recipes/marslab/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "marslab" %}
+{% set version = "0.9.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/marslab-{{ version }}.tar.gz
+  sha256: 2b4bd12019b307f22e691f3f3743d17846c7da76d65592a15b5c2f210937c42a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - astropy
+    - clize
+    - cytoolz
+    - dustgoggles
+    - fs
+    - hypothesis
+    - matplotlib-base
+    - more-itertools
+    - numpy
+    - pandas
+    - pathos
+    - pdr
+    - pytest
```

Comment:
I don't think `pytest` and other packages needed only for testing should be included here. The list should mainly be what's defined in:
https://github.com/MillionConcepts/marslab/blob/main/setup.py#L19-L29

and then it's a choice whether to include other extra dependencies.

Reply:
Removed pytest and pytest-cov